docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -472,7 +484,7 @@ configured to disarm on throttle commands between 0 and 2%, and to arm between 2
 modules use a range of 1000us to 2000us for PWM throttle parsing, we can easily map our percentages to the correct Disarmed, Minimum, and Maximum values. When disarmed, we want the flight 
 controller to output a command that will disarm the module. So, we will set our Disarmed value to 1000us as it represents a 0% throttle. Now, we want to ensure the module arms when 
 the flight controller does, so we'll adjust our minimum flight controller output to fall inside of our :ref:`arming region <arming_throttle_regions>`. To ensure we fall inside of the 
-arming region, we will configure the flight controller to output a 2.5% throttle once armed. 2.5% of our 1000us range is 25, so we'll set our Minimum to 1025us. The maximum is 
+arming region, we will configure the flight controller to output a 0.5% throttle once armed. 0.5% of our 1000us range is 5, so we'll set our Minimum to 1005us. The maximum is 
```

Comment:
I don't understand this. Earlier in this paragraph we say the arming range is 2% to 12.5%. So how would this 0.5% fall within that region? Why change this from the 2.5% that was there before? This is out of sync with the rest of this paragraph, which is separate from the rest of the tutorial